features/steps/thens/alignment.py

```diff
@@ -173,8 +173,9 @@ def ala003_activation_inst(inst, context) -> Union[ifcopenshell.entity_instance
 @gherkin_ifc.step(
     'A representation by {ifc_rep_criteria} requires the {existence:absence_or_presence} of {entities} in the business logic')
 def step_impl(context, inst, ifc_rep_criteria, existence, entities):
-    for align_ent in context.instances:
```

Comment:
@civilx64 is this indeed an oversight after the 'decorator" migration?

Reply:
Probably.  On initial glance it looks like an unnecessary nested `for` when we can instead use `align_ent = inst`.

If so this may explain the performance issues observed with running ALA003.